src/components/CatalogOverlay/CatalogOverlayComponent.tsx

```diff
@@ -203,7 +203,7 @@ export class CatalogOverlayComponent extends React.Component<WidgetProps> {
                 key={columnName} 
                 name={columnName} 
                 cellRenderer={(rowIndex, columnIndex) => (
-                    <Cell key={`cell_${columnIndex}_${rowIndex}`} interactive={true}>{coloumnData[rowIndex]}</Cell>
+                    <Cell style={{lineHeight: "250%"}} key={`cell_${columnIndex}_${rowIndex}`} interactive={true}>{coloumnData[rowIndex]}</Cell>
```

Comment:
would prefer this to be defined in CSS and for you to just use a `className`, since it is reused elsewhere.